root/static/scripts/common/i18n/expand2.js

```diff
@@ -172,11 +173,22 @@ function parseTextContent() {
   if (typeof text !== 'string') {
     return NO_MATCH;
   }
-  if (state.replacement !== NO_MATCH) {
-    /* flow-include if (state.replacement instanceof Symbol) throw 'impossible' */
-    text = text.replace(/%/g, he.encode(state.replacement));
```

Comment:
Obviously `text.replace` can only replace text with text, so we need to change this to split on '%' and replace the '%' in the list with our replacement.